packages/taco/test/test-utils.ts

```diff
@@ -222,6 +226,17 @@ export const testTimeConditionObj: TimeConditionProps = {
   chain: TEST_CHAIN_ID,
 };
 
+export const testJsonApiConditionObj = {
+  conditionType: JsonApiConditionType,
+  endpoint: 'https://api.coingecko.com/api/v3/simple/price',
```

Comment:
Does a request to this endpoint actually goes off during tests?

Reply:
no, that endpoint would only be hit at decryption (condition evaluation) time

Reply:
We should avoid calling external resources during tests. We can mock this request during evaluation and replace it with `https://_this_would_totally_fail.com` to make sure it crashes if we forget to use the mock etc

Reply:
it's never called during a test, it's just an example url to test the schema

Reply:
fixed in https://github.com/nucypher/taco-web/pull/550/commits/466fc939f4e89e13fd74a65b20fc6181910a64bc